app/views/layouts/application.html.erb

```diff
@@ -42,7 +43,7 @@
           </ul>
         </div>
       </div>
-    </nav>
+    </nav> -->
     <div class="container">
       <div class="row justify-content-center">
 
```

Comment:
for this flash message, we can use       

`
        <% flash.each do |type, msg| %>
          <div align="center">
            <div class="alert alert-<%=type%>">
              <%= msg  %>
            </div>
          </div>
        <% end %>
  `

and then use the [bootstrap alert classes](https://getbootstrap.com/docs/5.0/components/alerts/) to give colors based on the result of the flash